test/doro/world/marshal_test.exs

```diff
@@ -19,22 +19,29 @@ defmodule Doro.World.MarshalTest do
     } do
       ice = marshalled |> find_entity_by_id("ice")
       assert ice |> Map.get(:behaviors) == []
-      player = marshalled |> find_entity_by_id("_player")
-      assert MapSet.equal?(MapSet.new(player.behaviors), MapSet.new(["visible", "player"]))
-      god = marshalled |> find_entity_by_id("_god")
-      assert god |> Map.get(:behaviors) == ["god"]
+      player = marshalled |> find_entity_by_id("bathroom")
```

Comment:
take a look here and make sure this test is proving the right thing.  It used to read `with canonical behaviors` but now i think because the behaviors have `own` property, this really should only marshal own behaviors and all others would live on the prototype so just checking for the `proto` seems like enough.  Right?